cdhweb/people/models.py

```diff
@@ -178,7 +178,8 @@ def project_manager_years(self):
     def _current_position_query(self):
         # query to find a person with a current cdh position
         # person *has* a position and it has no end date or date after today
-        return models.Q(positions__isnull=False) & (
+        today = date.today()
+        return models.Q(positions__start_date__lte=today) & (
             models.Q(positions__end_date__isnull=True)
             | models.Q(positions__end_date__gte=date.today())
         )
```

Comment:
does this get used elsewhere? seems like it'd affect other things too (and glad we fixed it!)

Reply:
I think it gets used in multiple places, which is why we had a method for it. Yeah, not sure how we missed that one (although it's an edge case for sure)